napari/util/colormaps/colormaps.py

```diff
@@ -84,7 +84,7 @@ def _low_discrepancy_image(image, seed=0.5):
     """
     phi = 1.6180339887498948482
     image_out = (seed + image / phi) % 1
-    return image_out
+    return np.clip(image_out, 0.00001, 1.0-0.00001)
```

Comment:
Maybe add a comment here about why clipping is needed?

Reply:
added